tests/modules/sightings/resources/utils.py

```diff
@@ -70,25 +70,28 @@ def create_sighting(
 
 
 # not yet implemented
-def read_sighting(flask_app_client, user, enc_guid, expected_status_code=200):
+def read_sighting(flask_app_client, user, sight_guid, expected_status_code=200):
     with flask_app_client.login(user, auth_scopes=('sightings:read',)):
-        response = flask_app_client.get('%s%s' % (PATH, enc_guid))
+        response = flask_app_client.get('%s%s' % (PATH, sight_guid))
 
     assert isinstance(response.json, dict)
     assert response.status_code == expected_status_code
     if expected_status_code == 200:
-        assert response.json['id'] == str(enc_guid)
+        assert response.json['id'] == str(sight_guid)
     return response
 
 
-# not yet implemented
-def delete_sighting(flask_app_client, user, enc_guid, expected_status_code=204):
-    with flask_app_client.login(user, auth_scopes=('sighting:write',)):
-        response = flask_app_client.delete('%s%s' % (PATH, enc_guid))
+def delete_sighting(flask_app_client, user, sight_guid, expected_status_code=204):
+    with flask_app_client.login(user, auth_scopes=('sightings:write',)):
+        response = flask_app_client.delete('%s%s' % (PATH, sight_guid))
 
     if expected_status_code == 204:
         assert response.status_code == 204
     else:
         test_utils.validate_dict_response(
             response, expected_status_code, {'status', 'message'}
         )
+
+
+def row_count(db, cls):
```

Comment:
Really neat idea, I like it. 
But should be moved to tests/utils as it's really helpful and not sighting specific.

Reply:
i was thinking of that -- i just didnt know if it was nifty _enough_.  :)  i _almost_ made the more generic multi-object one the function ... but then wasnt sure if i ultimately would want to throw another class into the mix.  maybe we can move it to util and have the generic one accept a (optional?) list of classes to return counts on.   like you always get a list of 1-N counts.

Reply:
going to work on this presently.

Reply:
moved.